CHANGES.md

```diff
@@ -1 +1,42 @@
-## DC/OS 1.12.0
\ No newline at end of file
+## DC/OS 1.12.0
+
+```
+* For any significant improvement to DC/OS add an entry to Fixed and Improved section.
```

Comment:
How are we defining `significant`?

Reply:
We need to work on the guidelines. The idea is that all user-facing changes must have a corresponding changelog entry.

The idea is each individual pull request needs to justify why it should _not_ need to come with a changelog entry (it must then explain why the change is purely internal / not user-facing), instead of the other way round.

Reply:
We have to evolve the guidelines. We have to socialize the idea, understand the kind of situations we land into with respect to the addition of changelog entries, and then we could add this to the PR template.

Reply:
> How are we defining significant?

Our thinking is, features, bug fixes, bumps are significant.The final goal is about communicating what the customer can expect/see from this release. 

Things like fixing a test, refactors, documentation improvements, addition of tests _need not_ be included in the change-log. 

I was not thinking of explaining this,  I was thinking that it is understandable by the PR creator. 
The challenge I see is enforcing the process of requesting for change log when reviewing.Let's see how it goes.